Makefile

```diff
@@ -53,7 +53,7 @@ tools:
 
 presubmit: coverage fmt tools
 
-travis-presubmit: test fmt tools
```

Comment:
Can we please keep running tests as part of travis-presubmit? This is the only way we can know that a pull request passes all tests. Without it, we might merge one that breaks the master branch.